seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -119,8 +119,7 @@ avalanche_c,woofi,1,2022-11-08,0xae581cf151eacf9877f628e722514674f3f13b19abcce4e
 avalanche_c,woofi,1,2022-05-10,0x4ab2a9385bda8de7df444e6f342c12dd70ad6300fdf3a351d80f46e119a7811c,122,0x50b7545627a5162f82a992c33b87adc75187b218,4.00000076,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,126322.419554 
 avalanche_c,woofi,1,2022-07-28,0xa1bcc8d3ec39c8694c2be30f5489091fe1bfe8ffd570841e55e00bef6ee3359c,6,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0.008699999999999999,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,0.197657
 avalanche_c,woofi,1,2022-09-07,0xf48e0d36d6a51f82edecb408b20c1aa275fddc71f18d4118e734e3116c19d359,8,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,49.97,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,2.714758315778573433
-bnb,biswap,1,2022-11-22,0x66a602998a3585037eb498c103b009a4face2faf93ea57dfb1a44d1fa4a06dad,163,0xe9e7cea3dedca5984780bafc599bd69add087d56,33.830622552179025128,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,2128.2655
-bnb,biswap,1,2022-11-22,0x9050a1e2f7aa392e39195b5b22382d595e94104b72699655b198c498f82bbfe6,153,0xe9e7cea3dedca5984780bafc599bd69add087d56,26.146488901376957832,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,1643.319
-bnb,biswap,1,2022-11-22,0xe2efd633720bd29322da7e39857e98b3c9904a1c55f9b3995441160983e06296,119,0xe9e7cea3dedca5984780bafc599bd69add087d56,43.898605549510836795,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,2756.0281
-bnb,biswap,1,2022-11-22,0x0b9ba3ee4a0b3f4db2baba70be527f5a50aea77423cd921106ec9f0ac8dcf169,115,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.018248512111984361,0xe9e7cea3dedca5984780bafc599bd69add087d56,4.837104443141354752 
-bnb,biswap,1,2022-11-22,0xaf03946d9b98e5d6e841bb50555208801e48d773a8dfdf1ecc203d943c27c603,350,0x2170ed0880ac9a755fd29b2688956bd959f933f8,6.247796965743473757,0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c,0.43849
\ No newline at end of file
+bnb,fraxswap,1,2022-11-19,0x36f1b32a8115300906f74d3ddf2051d62a226045611a251d84a8a32395ead967,439,0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40,0.251513613449462899,0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c,0.001
+bnb,fraxswap,1,2022-10-26,0xe09dd465622225d25f5555214eb8e8e6e74d4a4fa5f6ac51b35f520150011f24,50,0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40,4.663005228579123357,0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c,0.018388924302368166
+ethereum,fraxswap,1,2022-07-25,0x36ea8f58b542aed8c3f35ce396da15054e04a522efb25281c8ed0080a081083a,200,0x853d955aCEf822Db058eb8505911ED77F175b99e,6321.896197119067345439,0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0,1000
+ethereum,fraxswap,1,2022-08-21,0x39714256ccf47a89e297b726e8de9eec9446949a224d5e2215fd4dc87c0c8969,406,0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0,2000.000000000000121322,0x853d955aCEf822Db058eb8505911ED77F175b99e,13014.187897251069443882
```

Comment:
GM @Kndlexi , do you think we can add some seeds for polygon fraxswap as well?

Reply:
Added now